REPOSITORY-GUIDELINES.md

```diff
@@ -20,13 +20,24 @@ Core repositories are all those located under the
 
 ### Core Repository Requirements
 
-- Repository must live under github.com/knative/project-name
-- Must adopt the Knative Code of Conduct
+- Repository must live under github.com/knative/project-name.
+- Must adopt the Knative Code of Conduct.
 - All code projects must use the Apache License version 2.0.
 - Documentation repositories must use Creative Commons License version 4.0.
 - All OWNERS must be members of the Knative community.
 - Repository creation must be approved by the Technical Oversight Committee.
 
+### Requirements for Repository Donation
+
+The Technical Oversight Committee may allow repositories to be donated to the
+Knative project. In *addition* to meeting the core repository requirements,
+donated repositories must:
+
+- Adopt the Google CLA bot.
+- All contributors to the donated repository must have signed the Google CLA.
+- Boilerplate text across all files should attribute copyright as follows:
+  `"Copyright <year> The Knative Authors"
+

```

Comment:
Maybe Add bullets about some technical  guidelines:
- Donated repo should be moved with commit history using the Github move features that everything get's transferred only thing new is the org/user as the ownership and URL location. If repo is not in github then the a new repo is created and the master branch push to it including history.

Reply:
I'm supportive of adding a guideline around bringing the full git repo history as part of the donation. Want to make sure we capture that as a "SHOULD" instead of a must however (as you have it written).

Reply:
Is there a request to change the text here?